src/data/complex/is_R_or_C.lean

```diff
@@ -346,7 +349,7 @@ by { have := I_mul_I_ax, tauto }
 begin
   by_cases h : (I : K) = 0,
   { simp [h] },
-  { field_simp [h], simp [mul_assoc, I_mul_I_of_nonzero h] }
+  { field_simp [h, mul_assoc, I_mul_I_of_nonzero h] }
```

Comment:
```suggestion
  { field_simp [mul_assoc, I_mul_I_of_nonzero h] }
```
Untested, but I believe that the change to `field_simp` a month or two ago made it unnecessary to mention "nonzero" hypotheses.